src/pathwayvis/components/reactionpanel/panelitem/panelitem.component.html

```diff
@@ -45,17 +41,46 @@
         </md-button>
       </md-list-item>
       <div ng-if="$ctrl.bounds" ng-repeat="item in $ctrl.items">
```

Comment:
This logic should not live in panelitem. I'm currently researching how to pass a component dynamically.